digid_eherkenning/saml2/base.py

```diff
@@ -177,6 +177,15 @@ def verify_saml2_response(self, response, client_ip_address):
                 OneLogin_Saml2_ValidationError.WRONG_INRESPONSETO,
             )
 
+        # I remember reading somewhere that the assurance level on the response
```

Comment:
should we create a ticket for this to have it on the backlog at least (of the library)?

Reply:
I put the remark there instead of the backlog, because I'm not sure it's actually needed. But iff it's needed, this is the place were someone will stumble upon the gotcha in the SAML spec.

Reply:
it reads a bit similar to #18, but I leave the decision to you if a ticket is warranted or not.